BIPs/00batched/authorizer/new-chain-template.json

```diff
@@ -9,35 +9,30 @@
       "stopAmplificationParameterUpdate()": "lm",
       "pause()": "emergency",
       "disable()": "emergency",
-      "enableRecoveryMode()": ["emergency", "PoolRecoveryHelper"],
+      "enableRecoveryMode()": ["emergency", "20221123-pool-recovery-helper"],
```

Comment:
Still not quite there yet

like "20221123-pool-recovery-helper/PoolRecoveryHelper"